koku/masu/external/kafka_msg_handler.py

```diff
@@ -313,6 +328,15 @@ def extract_payload(url, request_id, context={}):  # noqa: C901
     report_meta["account"] = schema_name[4:]
     report_meta["request_id"] = request_id
 
+    invalid_csvs = find_invalid_csvs(temp_dir)
```

Comment:
I think we might need to catch the Exception here for the case that the report type is invalid.  That way we can remove the temp_dir and not continue to process these invalid files.